temporian/core/serialize.py

```diff
@@ -0,0 +1,340 @@
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Serialization / unserialization of a processor."""
+
+from google.protobuf import text_format

```

Comment:
Is this a google internal tool? Doesn't seem to be available in our environment 👀

Reply:
The name is weird, but this is the classical protobuf python package (https://pypi.org/project/protobuf/#description).

Reply:
Import was failing but I reinstalled the env and it worked. My bad!